app/helpers/bookings_helper.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+module BookingsHelper
+  def check_status_booking(status)
+    if status.pending?
+      "primary"
+    else
+      status.confirmed? ? "success" : "danger"
+    end
```

Comment:
```suggestion
  def check_status_booking booking
    booking.pending? ?  :primary : booking.confirmed? ? :success : :danger
```
Thử như này có chạy không